README.md

```diff
@@ -24,6 +24,12 @@ On Windows
 
 There is no need to install Gradle to run the samples.
 
+## Offline sample data
+Some samples require offline data. The first time you refresh the Gradle project in an IDE, or run the `build` or 
+`run` tasks, the `downloadData` task will automatically download a local data folder called samples-data into the 
+directory with arcgis-java-runtime-samples. To get the latest data, delete the old `samples-data` folder, and then run
+`downloadData`.
```

Comment:
I ran the `run` task and the data did not get downloaded.  Prior to this there is no mention of refreshing your Gradle project or running the `build` task.